pkgs/applications/audio/easyabc/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, fetchFromGitHub, python39, makeWrapper, fluidsynth, soundfont-fluid, wrapGAppsHook, abcmidi, abcm2ps, ghostscript }:
+
+# requires python39 due to https://stackoverflow.com/a/71902541
+python39.pkgs.buildPythonApplication rec {
+  pname = "easyabc";
+  version = "1.3.8.6";
+
+  src = fetchFromGitHub {
+    owner = "jwdj";
+    repo = pname;
+    rev = "6461b2c14280cb64224fc5299c31cfeef9b7d43c";
+    sha256 = "sha256-leC3A4HQMeJNeZXArb3YAYr2mddGPcws618NrRh2Q1Y=";
```

Comment:
```suggestion
    hash = "sha256-leC3A4HQMeJNeZXArb3YAYr2mddGPcws618NrRh2Q1Y=";
```

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/237197/commits/5fa3e97bef6ce3b29331a1d0cfc006132d3c79ed